### DISCUSSION TITLE
Line break

### DISCUSSION THREAD

**Discussion by SandraEliz** [open-sdg/open-sdg#1866] [Q&A] [answered]
Hello
 
In the metadata section, can line breaks be added in this tag?
 
source_organization_1:
 
I want to put for example:
 
Source 1
Source 2
 
Separated with a line break
 
Thank you

**Comment by brockfanning** [comment, marked as answer]
@SandraEliz I think there are a couple of ways to do that.

One is to insert the `<br>` HTML tag into the content. For example:

`source_organisation_1: Source 1<br>Source 2`

Another way would be to use the `>-` approach and use actual line-breaks (I believe 2 are needed). For example:

```
source_organisation_1: >-
  Source 1


  Source 2
```

**Reply by SandraEliz** [reply]
Thanks
